test/testdrive/avro-nonnull-record.td

```diff
@@ -28,12 +28,14 @@ $ set writer-schema={
     ]
   }
 
-$ avro-ocf-write path=data.ocf schema=${writer-schema}
+$ kafka-create-topic topic=data
+$ kafka-ingest topic=data format=avro schema=${writer-schema}
```

Comment:
we should really make kafka-ingest have a `create-topic=true` field...

Reply:
Keeping them separate is intentional! There's a bunch of options on `kafka-create-topic` that would be a hassle to duplicate on `kafka-ingest`. It's only a few extra options.